diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/NewlinesRule.kt

```diff
@@ -441,13 +448,21 @@ class NewlinesRule(configRules: List<RulesConfig>) : DiktatRule(
             }
             WRONG_NEWLINES.warnAndFix(configRules, emitWarn, isFixMode,
                 warnText, node.startOffset, node) {
-                invalidCommas.forEach { comma ->
-                    val nextWhiteSpace = comma.treeNext.takeIf { it.elementType == WHITE_SPACE }
-                    comma.appendNewlineMergingWhiteSpace(nextWhiteSpace, nextWhiteSpace?.treeNext ?: comma.treeNext)
+                invalidCommas.forEach { commaOrRpar ->
+                    val nextWhiteSpace = commaOrRpar.treeNext?.takeIf { it.elementType == WHITE_SPACE }
+                    if (commaOrRpar.elementType == COMMA) {
+                        nextWhiteSpace?.treeNext?.let {
+                            commaOrRpar.appendNewlineMergingWhiteSpace(nextWhiteSpace, nextWhiteSpace.treeNext)
+                        } ?: commaOrRpar.treeNext?.treeParent?.appendNewlineMergingWhiteSpace(nextWhiteSpace, commaOrRpar.treeNext)
```

Comment:
This looks a little tricky (accessing `treeParent`, but only if `treeNext != null`?). Can you provide some comments here?